src/mono/browser/runtime/package-lock.json

```diff
@@ -1,130 +1,203 @@
 {
   "name": "@microsoft/dotnet-runtime",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 3,
```

Comment:
Make sure you actually intended to update the lockfile, lockfile changes get generated at random in my experience

Reply:
if we want to stay on Node14 this must stay `"lockfileVersion": 1`
I would appreciate if that could be separate PR anyway.

Reply:
this part was reverted